plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller-device.c

```diff
@@ -449,6 +456,7 @@ fu_logitech_bulkcontroller_device_write_firmware(FuDevice *device,
 		g_prefix_error(error, "error in writing start transfer packet: ");
 		return FALSE;
 	}
+	fu_progress_step(progress);
```

Comment:
with this new way of doing it, would it make sense to get into a habit of commenting where it's supposed to go?  It might make tracking down mismatches easier later?
```
 fu_progress_step(progress); /* erase */
 fu_progress_step(progress); /* write */
 fu_progress_step(progress); /* busy */
```

Reply:
we could do, but the fu_progress_step() signifies the *end* of the action. So maybe `fu_progress_step(progress); /* busy done */` -- or maybe `fu_progress_step()` is the wrong verb, perhaps `fu_progress_step_done()` or `fu_progress_done()` is more obvious.

Reply:
Yeah I think `step_done` makes more sense.

Reply:
I changed it to `step_done`.